Jenkinsfile.release

```diff
@@ -74,85 +78,95 @@ podTemplate(cloud: 'openshift', label: pod_label, yaml: pod) {
+                        ore gcloud promote-image \
+                            --log-level=INFO \
+                            --project=\${gcp_project} \
+                            --json-key \${GCP_IMAGE_UPLOAD_CONFIG} \
+                            --family fedora-coreos-${params.STREAM} \
+                            --image "${gcp_image}"
+                        """)
+                    }
+                }
             }
-        }
 
-        if (params.AWS_REPLICATION == 'true') {
-            // Replicate the newly uploaded AMI to other regions. Intentionally
-            // split out from the 'Upload AWS' stage to allow for tests to be added
-            // at a later date before replicating said image.
-            //
-            // We have to re-run the coreos-meta-translator as aws-replicate
-            // only modifies the meta.json
-            stage('Replicate AWS AMI') {
-                shwrap("""
-                export AWS_CONFIG_FILE=\${AWS_FCOS_BUILDS_BOT_CONFIG}
-                cosa aws-replicate --build=${params.VERSION} --log-level=INFO
-                cosa generate-release-meta --build-id ${params.VERSION} --workdir .
-                cosa buildupload --build=${params.VERSION} --skip-builds-json s3 --acl=public-read ${s3_stream_dir}/builds
-                """)
+
+            if ((basearch in ['x86_64', 'aarch64']) && params.AWS_REPLICATION == 'true') {
+                // Replicate the newly uploaded AMI to other regions. Intentionally
+                // split out from the 'Upload AWS' stage to allow for tests to be added
+                // at a later date before replicating said image.
+                //
+                // We have to re-run the coreos-meta-translator as aws-replicate
+                // only modifies the meta.json
+                stage('Replicate ${basearch} AWS AMI') {
+                    shwrap("""
+                    export AWS_CONFIG_FILE=\${AWS_FCOS_BUILDS_BOT_CONFIG}
+                    cosa aws-replicate --build=${params.VERSION} --arch=${basearch} --log-level=INFO
+                    """)
+                }
             }
         }
 
         stage('Publish') {
-          // Run plume to publish official builds; This will handle modifying
-          // object ACLs, modifying AMI image attributes,
-          // and creating/modifying the releases.json metadata index
-          shwrap("""
-          export AWS_CONFIG_FILE=\${AWS_FCOS_BUILDS_BOT_CONFIG}
-          plume release --distro fcos \
-              --version ${params.VERSION} \
-              --stream ${params.STREAM} \
-              --bucket ${s3_bucket}
-          """)
+            // generate the release meta data and upload it back to s3
+            shwrap("""
+            cosa generate-release-meta --build-id ${params.VERSION} --workdir .
+            cosa buildupload --build=${params.VERSION} --skip-builds-json s3 --acl=public-read ${s3_stream_dir}/builds
+            """)
```

Comment:
Minor: space?

Reply:
not following.. you want a blank line to follow this code block?